staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -898,6 +875,40 @@ func (s *store) GetList(ctx context.Context, key string, opts storage.ListOption
 	return s.versioner.UpdateList(listObj, uint64(withRev), continueValue, remainingItemCount)
 }
 
+func (s *store) processListItem(ctx context.Context, aggregator ListErrorAggregator, predicate storage.SelectionPredicate, v reflect.Value, newItemFunc func() runtime.Object, kv *mvccpb.KeyValue) (bool, error) {
```

Comment:
How someone reading this function for the first time would know what the returned bool is supposed to mean?